PowerPointLabs/PowerPointLabs/ActionFramework/Action/ColorsLabActionHandler.cs

```diff
@@ -7,7 +7,7 @@ namespace PowerPointLabs.ActionFramework.Action
     [ExportActionRibbonId("ColorsLabButton")]
     class ColorsLabActionHandler : ActionHandler
     {
-        protected override void ExecuteAction(string ribbonId)
```

Comment:
I thought I might need the ribbon tag for some functions but in the end I did not use it. I will remove it.